projects/kubernetes/boot.sh

```diff
@@ -16,26 +16,36 @@ set -e
 : ${KUBE_EFI:=}
 : ${KUBE_MAC:=}
 : ${KUBE_CLEAR_STATE:=}
+#: ${KUBE_MASTER_AUTOINIT:=}
+external_data=${KUBE_METADATA}
 
 [ "$(uname -s)" = "Darwin" ] && KUBE_EFI=1
 
 suffix=".iso"
 [ -n "${KUBE_EFI}" ] && suffix="-efi.iso" && uefi="--uefi"
 
+data={
+if [ -n "${external_data}" ] ; then
+	if [ -s "${external_data}" ] ; then
+		external_data=$(tr '\n' ' ' <$external_data | tr -d '\t' )
+	fi
+	data+=$external_data
```

Comment:
Is this `+=` a bashism? I can't find it described in http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html. Note that this script uses `/bin/sh`, `dash` is a more POSIXly correct shell and:
```
$ dash
$ foo="1"
$ foo+="1"
dash: 2: foo+=1: not found
```